backend/ERPNumber1/ERPNumber1/Program.cs

```diff
@@ -13,6 +13,22 @@
 // Add services to the container.
 builder.Services.AddControllers();
 
+// Add CORS
+builder.Services.AddCors(options =>
+{
+    options.AddPolicy("AllowFrontend", policy =>
+    {
+        policy.WithOrigins(

```

Comment:
This is gonna be a big problem when deploying to production. These URL's should be retrieved from an environment variable (.env file) so they can be set during deployment

Reply:
I see that the line in question isn't included in the snippet. I of course meant the array starting on line 22